source/_integrations/phone_modem.markdown

```diff
@@ -0,0 +1,73 @@
+---
+title: Phone Modem (Formerly Modem Caller ID)
+description: Instructions on how to integrate the Caller ID sensor into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: '0.40'
+ha_iot_class: Local Polling
+ha_domain: phone_modem
+---
+
+The `phone_modem` integration uses an available modem for collecting caller ID information. It requires a Hayes AT compatible modem that supports caller ID detection (via AT+VCID=1). Usually any modem that uses a CX93001 will support this.
+
+When the sensor detects a new call, its state changes to 'ring' for each ring and 'callerid' when caller id information is received. It returns to 'idle' once ringing stops. The state event includes an attribute payload that includes the time of the call, name and number.
+
+This integration also offers two services. `phone_modem.reject_call` to pick up and then hang up the call to properly reject a call (via ATA anf ATH). `phone_modem.hangup_call` to hang up an existing call (via ATH).
+
+## Setup
+
+To find the path of your USB modem, run:
+
+```bash
+ls /dev/ttyACM*
+```
+
+If Home Assistant (`hass`) runs with another user (e.g., `homeassistant`) give access to the stick with:
+
+```bash
+sudo usermod -a -G dialout homeassistant
+```
+
+Depending on what's plugged into your USB ports, the name found above may change. You can lock in a name, such as `/dev/modem`, by following [these instructions](http://hintshop.ludvig.co.nz/show/persistent-names-usb-serial-devices/).
```

Comment:
These instructions are specifically for users of Home Assistant Core installation types. Move this to the bottom of the page, and wrap it in a `{% details %}` block.

Reply:
Sure thing, I kept it on the top because I wanted users to first see how they could get the device path for the modem in case it was not the default /dev/ttyACM0. I will move it as you like.